react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -378,7 +390,8 @@ public void attachToContainer(ViewGroup viewGroup, OnMediaLibraryButtonListener
                                   OnEditorAutosaveListener onEditorAutosaveListener,
                                   OnAuthHeaderRequestedListener onAuthHeaderRequestedListener,
                                   RequestExecutor fetchExecutor,
-                                  OnImageFullscreenPreviewListener onImageFullscreenPreviewListener) {
+                                  OnImageFullscreenPreviewListener onImageFullscreenPreviewListener,
+                                  OnMediaEditorListener onMediaEditorListener) {
```

Comment:
We also need a PR to WPAndroid that adds this new parameter where [`GutenbergContainerFragment` calls this method](https://github.com/wordpress-mobile/WordPress-Android/blob/df83d585f5a70442bf718b46e4236b064a3f0d0d/libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergContainerFragment.java#L57). Otherwise, merging this will break building Gutenberg from source on WPAndroid (i.e., building with `wp.BUILD_GUTENBERG_FROM_SOURCE=true` added to WPAndroid's `gradle.properties` file).

Reply:
Done in https://github.com/wordpress-mobile/WordPress-Android/pull/11175